app/components/Modals/BaseModal/BaseModal.scss

```diff
@@ -40,7 +40,7 @@
 
   svg {
     path {
-      fill: var(--panel-full-height-nav-background);
+      fill: var(--panel-full-height-header-icon-color);
```

Comment:
Also realized we are using the wrong color here in 2.0.0 (no issue currently open)